packages/x-data-grid/src/constants/cssVariables.ts

```diff
@@ -77,19 +77,19 @@ const keys = {
   },
   transitions: {
```

Comment:
I'm ok with the `vars` object keys remaining plural, but equally happy to update these if others think they should be singular too.